packages/test-vectors/src/sidetree-universal-wallet/wallet-keypair.json

```diff
@@ -9,15 +9,13 @@
         "publicKeyJwk": {
           "crv": "Ed25519",
           "x": "BnamLr9mV2goDx4gfFzaBCBXWy2Bpa64cNNhVMIe4vc",
-          "kty": "OKP",
-          "kid": "DrUSp0MnN8txc0J-odEXDRu-QRRDpOP9c2awKGW2Ozk"
+          "kty": "OKP"
```

Comment:
Also need to update the code that generated those fixtures, in `core`

Reply:
these fixtures are generated by the wallet package, not core. I will check core, but should be seeing test failures if the was fixture drift.

Reply:
tested locally, there are failures, I will update the fixtures.

Reply:
Looks like core tests are failing https://github.com/transmute-industries/sidetree.js/runs/1298261123#step:9:248

Reply:
yep, thats a good sign, ci is catching fixture drift.